internal/cmd/price/command_list.go

```diff
@@ -255,20 +255,14 @@ func filterTable(table map[string]*ccloudv1.UnitPrices, filters []string, metric
 	return filteredTable, nil
 }
 
-func getKeys(m map[string]string) []string {
-	var slice []string
-	for key := range m {
-		slice = append(slice, key)
-	}
+func getSortedKeys(m map[string]string) []string {
```

Comment:
I think we should put this in utils too even though it's currently only used here, because it may be useful elsewhere later on and someone could end up rewriting it after not finding it in utils.